src/assets/server/rest/v1/schemas/common/verify-tenant-id-subdomain.json.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Name of the file is wrong: verify-tenant-id-subdomain.json.json
and $id still named 'auth-verify-tenant-redirect' instead 'verify-tenant-id-subdomain'
Pay attention!!!